### PR TITLE
[BD-46] feat: breadcrumb dark variant and mobile orientation

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Update mobile variant for Breadcrumb
- Implement dark variant of Breadcrumb
- Update Breadcrumb link color

Preview: https://deploy-preview-926--paragon-edx.netlify.app/

JIRA: 
https://openedx.atlassian.net/browse/PAR-439
https://openedx.atlassian.net/browse/PAR-581
https://openedx.atlassian.net/browse/PAR-582

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vlasovmichael! I've created [BLENDED-1040](https://openedx.atlassian.net/browse/BLENDED-1040) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

File: src/Breadcrumb/index.jsx [hunk 1]
```diff
@@ -48,12 +49,18 @@ Breadcrumbs.propTypes = {
...
+  /** The `Breadcrumbs` mobile variant view. */
+  isMobile: PropTypes.bool,
 };
 
 Breadcrumbs.defaultProps = {
   activeLabel: undefined,
   spacer: undefined,
   clickHandler: undefined,
+  variant: 'light',
+  isMobile: undefined,
```
Comment (monteri) [inline src/Breadcrumb/index.jsx:63]: Although it works good, is it more appropriate to set `isMobile` to `false` by default instead of `undefined`?

Comment (vlasovmichael) [reply src/Breadcrumb/index.jsx]: Fixed. Thanks!

**Review by adamstankiewicz — APPROVED** [review]

**Comment by edx-webhook** [comment]
@vlasovmichael 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/paragon&entry.867055334=https://github.com/edx/paragon/pull/926&entry.1484655318=https://github.com/vlasovmichael&entry.752974735=2021-12-07+13:52&entry.1917517419=2021-12-10+14:12&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**